### PR TITLE
Self surgery and other fixes

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl:
add: Self surgery
add: Test dummy spawn admin verb
add: Mecha race
tweak: Gib animation tweaked
tweak: Cryo tweaked
tweak: Speeder car tweaked
/:cl:

### CODE REVIEW

**Comment by tortellinitony** [comment]
fixes #44 fixes #56

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -839,8 +839,6 @@
 			dat += "<td width='20%'><a href='?src=\ref[src];jobban3=borer;jobban4=\ref[M]'>Borer</a></td>"
 
 		dat += "</tr></table>"
-		usr << browse(dat, "window=jobban2;size=800x450")
```
Comment (j-mie) [inline code/modules/admin/topic.dm:842]: ???

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm [hunk 2]
```diff
@@ -124,7 +124,7 @@
 			if(C && user.pulling == C && !C.buckled && !C.has_buckled_mobs() && !occupant)
 				var/turf/prevloc = C.loc
 				user.visible_message("<span class='danger'>[user] stuffs [C] into the gibber!</span>")
-				C.forceMove(src)
+//				C.forceMove(src)
```
Comment (j-mie) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm:127]: ???

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (tortellinitony) [reply code/modules/admin/topic.dm:842]: this makes it so the catban button shows up

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm (hunk 2 above)
Comment (tortellinitony) [reply code/modules/food_and_drinks/kitchen_machinery/gibber.dm:127]: that was causing issues with the animation

File: code/modules/mob/living/carbon/human/status_procs.dm [hunk 3]
```diff
@@ -3,11 +3,15 @@
...
 	update_health_hud()
+	sleep(amount)
+	update_health_hud()
 
 /mob/living/carbon/human/Weaken(amount, updating = 1, ignore_canstun = 0)
 	amount = dna.species.spec_stun(src,amount)
 	..()
 	update_health_hud()
+	sleep(amount)
+	update_health_hud()
```
Comment (MrsTonedOne) [inline code/modules/mob/living/carbon/human/status_procs.dm:14]: pahahahahahaha

File: tgui/src/interfaces/cryo.ract [hunk 4]
```diff
@@ -51,9 +51,9 @@
   <ui-section label='Temperature'>
 		<span class='{{temperatureStatus(adata.cellTemperature)}}'>{{Math.round(adata.cellTemperature)}} K</span>
 	</ui-section>
-	<ui-section label='Door'>
-    <ui-button icon='{{data.isOpen ? "unlock" : "lock"}}' action='door'>{{data.isOpen ? "Open" : "Closed"}}</ui-button>
-	</ui-section>
+  <ui-section label='Door'>
+		<ui-button icon='{{data.isOpen ? "unlock" : "lock"}}' action='door'>{{data.isOpen ? "Open" : "Closed"}}</ui-button>
+    </ui-section>
```
Comment (MrsTonedOne) [inline tgui/src/interfaces/cryo.ract:56]: \>changed a tgui interface

\>didn't recompile tgui

File: code/modules/mob/living/carbon/human/status_procs.dm [hunk 5]
```diff
@@ -3,11 +3,15 @@
...
 
 /mob/living/carbon/human/Weaken(amount, updating = 1, ignore_canstun = 0)
 	amount = dna.species.spec_stun(src,amount)
 	..()
 	update_health_hud()
+	sleep(amount)
```
Comment (Thunder12345) [inline code/modules/mob/living/carbon/human/status_procs.dm:13]: WHY

Comment (JohnGinnane) [reply code/modules/mob/living/carbon/human/status_procs.dm:13]: Yes, why

**Comment by cacogen** [comment]
Really need to make more specific changelogs in future. "Tweaked cryo" doesn't tell people much.

File: tgui/src/interfaces/cryo.ract (hunk 4 above)
Comment (tortellinitony) [reply tgui/src/interfaces/cryo.ract:56]: @MrStonedOne How does one recompile tgui exactly?

Comment (MrsTonedOne) [reply tgui/src/interfaces/cryo.ract:56]: step one, section off about 3 hours of time for this task.

step two, install node.js

step three, run install dependency.bat

step four... cryptic error message

step five google error message

step six, hold funeral for the last of the your hair to fall out from stress

step seven: it magically works properly when trying it.

step eight, run compile assets.bat.

step nine: see step four

**Pull request merged**